cmd/cli/plugin/package/package_install.go

```diff
@@ -34,7 +34,7 @@ func init() {
 	packageInstallCmd.Flags().StringVarP(&packageInstallOp.PackageName, "package-name", "p", "", "Name of the package to be installed")
 	packageInstallCmd.Flags().StringVarP(&packageInstallOp.Version, "version", "v", "", "Version of the package to be installed")
 	packageInstallCmd.Flags().BoolVarP(&packageInstallOp.CreateNamespace, "create-namespace", "", false, "Create namespace if the target namespace does not exist, optional")
-	packageInstallCmd.Flags().StringVarP(&packageInstallOp.Namespace, "namespace", "n", "default", "Namespace indicates the location of the repository from which the package is retrieved")
+	packageInstallCmd.Flags().StringVarP(&packageInstallOp.Namespace, "namespace", "n", getDefaultNameSpace(), "Namespace indicates the location of the repository from which the package is retrieved")
```

Comment:
Having to do this everywhere points to a posiblity of moving the definition of this flag to a higher level where it only gets executed once. All the files are part of the main package so we shoudl be able to move it up

Reply:
Best to leave as is since it will only make the code harder to read and there might be situations where a particulare subcommand wants a different behavior.  either way spelling it out in every case makes it more clear.